vmm/src/memory_manager.rs

```diff
@@ -1152,6 +1152,12 @@ impl MemoryManager {
 
         let region = self.add_ram_region(start_addr, size)?;
 
+        // Add region to the list of regions associated with the default
+        // memory zone.
+        if let Some(memory_zone) = self.memory_zones.get_mut(DEFAULT_MEMORY_ZONE) {
```

Comment:
Adjust first line of commit message "vmm: memory_manager: Add ACPI hotplug region to default memory zone"